docs/container.md

```diff
@@ -0,0 +1,65 @@
+# Container Instrumentation
+
+This document details the steps required to instrument a Java based container to use the opentracing [specialagent](https://github.com/opentracing-contrib/java-specialagent).
+
+## Introduction
+
+The steps below explain how to modify the Dockerfile for a Java application to automatically instrument your application using the opentracing specialagent.
```

Comment:
This page explains how to modify the Dockerfile (?why not Docker file?) for a Java application to instrument your application using the OpenTracing Special Agent. 
??What's the correct capitalization for this? It feels like a name. And, OpenTracing, not opentracing for sure.